### PR TITLE
Add reference to service key.

### PR DESCRIPTION
**Pull request by stevenpack**
Got advice from ex-Megaport AE that this was missing.

### CODE REVIEW

File: products/network-interconnect/src/content/partners/megaport.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ pcx-content-type: tutorial
 
 ## Create a VXC with a service key
 
-Once the service key is created and shared, anybody with the key can create a connection.
+Once your Cloudflare account team creates the service key and shares with you, anybody with the key can create a connection.
```
Comment (dcpena) [inline products/network-interconnect/src/content/partners/megaport.md:11]:
```suggestion
After the Cloudflare account team creates the service key and shares it with you, anybody with the key can create a connection.
```

Comment (dcpena) [reply products/network-interconnect/src/content/partners/megaport.md:11]: Small suggestion that I think will make this a little clearer for users

**Comment by stevenpack** [comment]
@dcpena would appreciate your input on this one Denise. Want to make sure it's line with standards, but got this feedback that it wasn't clear to customers *how* get they a service key.

File: products/network-interconnect/src/content/partners/megaport.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ pcx-content-type: tutorial
...
 
 1. Log in to the [Megaport Portal](https://portal.megaport.com/).
 1. Find the Megaport you want to use for the connection and click **+Connection**.
-1. Under **Choose Destination Type**, click **Enter Service Key**.
+1. Under **Choose Destination Type**, click **Enter Service Key**. Your Cloudflare account team will provide the Service Key via email.
```
Comment (dcpena) [inline products/network-interconnect/src/content/partners/megaport.md]: I deleted my previous suggestion after thinking about it some more. I think it makes sense to add the line about receiving the service key via email to the sentence before the procedure.

File: products/network-interconnect/src/content/partners/megaport.md (hunk 1 above)
Comment (dcpena) [inline products/network-interconnect/src/content/partners/megaport.md]:
```suggestion
The Cloudflare account team creates the Service Key and sends it to you via email. After the Service Key is sent, anyone who has the key can use it to create a connection.
```

**Review by dcpena — COMMENTED** [review]
> @dcpena would appreciate your input on this one Denise. Want to make sure it's line with standards, but got this feedback that it wasn't clear to customers _how_ get they a service key.

Happy to help! I made a suggestion to what you added, and if you agree with the wording I suggested, you can click the **Commit suggestion** button to accept and commit that wording.

File: products/network-interconnect/src/content/partners/megaport.md (hunk 2 above)
Comment (dcpena) [inline products/network-interconnect/src/content/partners/megaport.md]:
```suggestion
1. Under **Choose Destination Type**, click **Enter Service Key**.
```

**Review by KimJ15 — APPROVED** [review]

**Comment by KimJ15** [comment]
@stevenpack @dcpena These comments look good to me. Do you think we can merge?

**Comment by dcpena** [comment]
@KimJ15 I committed the pending comments I had so I'm good with merging this.

**Pull request merged**